packages/web-components/src/components.d.ts

```diff
@@ -393,6 +393,20 @@ export namespace Components {
          */
         "required"?: boolean;
     }
+    interface VaIcon {
+        /**
+          * the name of the icon to use
+         */
+        "icon": string;
+        /**
+          * the size variant of the icon
```

Comment:
Can we capitalize the documentation comments for each of the props? This text is what will be displayed in Storybook for public facing docs.

Reply:
done!